common/autoware_kalman_filter/include/autoware/kalman_filter/kalman_filter.hpp

```diff
@@ -0,0 +1,214 @@
+// Copyright 2018-2019 Autoware Foundation
```

Comment:
I think it's better. The same applies to the following files as well.

```suggestion
// Copyright 2024 Autoware Foundation
```

Reply:
I'm sorry it should be 2025. However, since it seems better to replace these mechanically, including other packages, I will put it on hold for now.